conda_env/specs/jinja2_file.py

```diff
@@ -0,0 +1,36 @@
+try:
+    import jinja2
+except ImportError:
+    jinja2 = None
+from .. import env
+from ..exceptions import EnvironmentFileNotFound, Jinja2NotInstalled
+
+
+class Jinja2Spec(object):
+    _environment = None
+
+    def __init__(self, filename=None, **kwargs):
+        self.filename = filename
+        self.msg = None
+
+    def can_handle(self):
```

Comment:
mmh you are right about that, maybe we can think on some sort of proxy. That would be for another PRI guess